src/components/BooksForm.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styles from '../css/modules/BooksForm.module.css';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const SelectCategories = () => {
+  const options = categories.map(category => (
+    <option key={category} value={category}>
+      {category}
+    </option>
+  ));
+
+  return (
+    <select name="categories" id="category">
+      <option key="placeholder" value="">
+        Category
+      </option>
+      {options}
+    </select>
+  );
+};
```

Comment:
These two things could be one. and usually you want the first category item to be the first in a select/option.

Reply:
I think it's a personal preference for me. I don't want an already chosen option so the default is an empty value.